components/sbm-core/src/main/java/org/springframework/sbm/build/impl/OpenRewriteMavenBuildFile.java

```diff
@@ -669,16 +669,17 @@ public void excludeDependencies(List<Dependency> excludedDependencies) {
 
     @Override
     public void addRepository(RepositoryDefinition repository) {
-        AddMavenRepository addMavenRepository = new AddMavenRepository(repository);
-        apply(addMavenRepository);
+        Recipe addRecipe = new AddMavenRepository(repository).doNext(new RefreshPomModel());
```

Comment:
Why do we need to refresh the model? This should not be required with the latest rewrite version.